web/packages/teleterm/src/ui/boot.tsx

```diff
@@ -32,8 +32,9 @@ async function boot(): Promise<void> {
     const appContext = new AppContext(globals);
 
     window.addEventListener('error', event => {
-      console.error(event.error.stack);
-      logger.error(event.error.stack);
+      const message = event.error ? event.error.stack : event.message;
```

Comment:
I added it because `ResizeObserver` throws the following error from time to time:
```
ResizeObserver loop limit exceeded
```
From what I see [it is not dangerous](https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded), but if I have time I will look at it, maybe we can fix it.

Reply:
How does this code takes care of this problem? Let's document why `error` can be missing in general terms. Does it boil down to `ErrorEvent` vs `Event`? https://developer.mozilla.org/en-US/docs/Web/API/Window/error_event#event_type

Reply:
Ah sorry, this of course doesn't take care of the problem, my comment was not precise.

I had to change it because the listener was failing with an error 'cannot read stack of undefined'. 
I think it happened because of the difference between `ErrorEvent` and `Event`. I added an explanation.